src/client/components/MovieForm.jsx

```diff
@@ -1,28 +1,67 @@
 import { useState } from "react";
 
-export default function MovieForm({ handleSubmit }) {
-    const [movie, setMovie] = useState({ title: '', description: '', runtimeMins: 60 });
+export default function MovieForm({ handleSubmit, movies }) {
+  const [movie, setMovie] = useState({
+    title: "",
+    description: "",
+    runtimeMins: 60,
+  });
 
-    const handleSubmitDecorator = (e) => {
-        e.preventDefault();
-        handleSubmit(movie);
-    }
+  const handleSubmitDecorator = (e) => {
+    e.preventDefault();
+    handleSubmit(movie);
+  };
 
-    const handleChange = (e) => {
-        const { name, value } = e.target;
+  const handleChange = (e) => {
+    const { name, value } = e.target;
 
-        setMovie({
-            ...movie,
-            [name]: name === 'runtimeMins' ? parseInt(value) : value
-        });
-    }
+    setMovie({
+      ...movie,
+      [name]: name === "runtimeMins" ? parseInt(value) : value,
```

Comment:
Ok - but ternaries are quite brittle. This is likely to change (more conditions), and the current implementation doesn't easily support change.

Ternaries are great for conditions that are only either or, with very low probability of more options.

Reply:
Refactored this line to use a simple OR operator instead of a ternary ✔️